frontend/src/components/login/Login.scss

```
@@ -0,0 +1,54 @@
+@import "../../constants.scss";
+
+body {
+  align-items: center;
+  text-align: center;
+}
+
+.admin-message {
+  color: #025e73;
+  margin-top: 60px;
+}
+
+.form {
```

This className should be more specific because we have ".form" in the user creation page currently and there are a few more forms on the site... It could be something like .login-form

```suggestion
.login-form
```

---

Thanks Jack, but will wait on this, because if we need a separate form.css then this may have to be rechanged. We can discuss on this.